bb8/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "bb8"
-version = "0.8.0"
-description = "Full-featured async (tokio-based) connection pool (like r2d2)"
+version = "0.8.0-rc.1"
```

Comment:
I won't be able to merge this PR with this stuff in it, please revert these changes.